bokeh/charts/builder/area_builder.py

```diff
@@ -174,13 +151,13 @@ def draw(self):
 
         Takes reference points from the data loaded at the ColumnDataSource.
         """
-        colors = self._set_colors(self.attr)
+        colors = cycle_colors(self._attr)
```

Comment:
cycle_colors should be called passing the palette (or you thought of doing this on a separate PR?)